tests/subsys/tracing/tracing_sys/src/main.c

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright (c) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <ztest.h>
+#if defined(CONFIG_TRACING_BACKEND_UART) || defined(CONFIG_TRACING_BACKEND_USB)
+#include "../../../../subsys/tracing/include/tracing_backend.h"
```

Comment:
Can we make it shorter, just like "include/tracing_backend.h" or something like this?

Reply:
It doesn't work if I check it, maybe need some time to take a look at this, thanks.